CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -22,5 +22,8 @@ public class Employee
         [ForeignKey(nameof(Company))]
```

Comment:
Field `salary` should be a properly cased property with proper encapsulation and typed appropriately (e.g., `decimal`). It must also be decorated with validation or database annotations if required.

```suggestion
public decimal Salary { get; set; }
```